qadence/ml_tools/models.py

```diff
@@ -181,7 +181,7 @@ def _transform_x(self, x: dict[str, torch.Tensor] | Tensor) -> dict[str, Tensor]
 
         else:
             assert isinstance(self.model, torch.nn.Module) and isinstance(x, Tensor)
-            return self._input_scaling * (x + self._input_shifting)
+            return self._input_scaling * x + self._input_shifting
```

Comment:
why do we want to make this change? this will break backwards compatibility...

Reply:
readded!